conftest.py

```diff
@@ -29,3 +33,31 @@ def enable_console_logging():
 def csrf_check_client() -> Client:
     """A Django test client instance that enforces CSRF checks."""
     return Client(enforce_csrf_checks=True)
+
+
+@pytest.fixture()
+def root_page() -> Page:
+    """Returns the root page. Useful for adding pages in tests."""
+    return Page.objects.filter(depth=1).get()
+
+
+@pytest.fixture()
+def home_page() -> HomePage:
+    """Returns the home page."""
+    return HomePage.objects.first()
+
+
+@pytest.fixture()
+def release_calendar_index() -> ReleaseCalendarIndex:
+    """Returns the release calendar index page, which is created via a migration."""
+    return ReleaseCalendarIndex.objects.first()
```

Comment:
Thought: Currently there's no constraint that only one of these exists.

- Should there be a constraint, or
- Should the fixture be a factory instead?

Reply:
There's one index per homepage.